generate_fixtures.py

```diff
@@ -129,6 +129,19 @@ def __init__(self):
         self.write_and_publish_repository(export_client=True)
 
 
+class TUFTestFixtureAttackRollback(TUFTestFixtureSimple):
```

Comment:
My Python-fu is not strong enough to offer serious review on this, maybe @davidstrauss should glance at it? On its face, it looks okay to me.

Reply:
This is a little more ad hoc than I'd prefer long-term; I'd really like to have high-level ways of specifying "attacks." At the same time, I think it's premature to expect to know these patterns yet. I'd like to more forward with this implementation and factor the "attack" fixtures into more descriptive, high-level approaches after we have a few.